content/en/docs/releases/supported-releases/index.md

```diff
@@ -51,15 +51,19 @@ current `<minor>` release. A patch is usually a small change relative to the `<m
 
 ## Support status of Istio releases
 
-| Version         | Currently Supported   | Release Date        | End of Life          | Supported Kubernetes Versions | Untested, but may work on Kubernetes Versions |
-|-----------------|-----------------------|---------------------|----------------------|-------------------------------|-----------------------------------------------|
-| master          | No, development only  |                     |                      |                               |                                               |
-| 1.10            | Yes                   | May 18, 2021        | ~Nov 2021 (Expected) | 1.18, 1.19, 1.20, 1.21        | 1.16, 1.17                                    |
-| 1.9             | Yes                   | February 9, 2021    | Aug 18, 2021         | 1.17, 1.18, 1.19, 1.20        | 1.15, 1.16                                    |
-| 1.8             | Yes                   | November 10, 2020   | May 12, 2021         | 1.16, 1.17, 1.18, 1.19        | 1.15                                          |
-| 1.7             | No                    | August 21, 2020     | Feb 25, 2021         | 1.16, 1.17, 1.18              | 1.15                                          |
-| 1.6             | No                    | May 21, 2020        | November 23, 2020    | 1.15, 1.16, 1.17, 1.18        |                                               |
-| 1.5 and earlier | No                    |                     |                      |                               |                                               |
+| Version         | Currently Supported   | Release Date        | End of Life          | Supported Kubernetes Versions | Tested, but not supported |
+|-----------------|-----------------------|---------------------|----------------------|-------------------------------|---------------------------|
+| master          | No, development only  |                     |                      |                               |                           |
+| 1.11            | Yes                   | August 2021 (Expect)| ~Feb 2022 (Expected) | 1.19, 1.20, 1.21, 1.22        | 1.16, 1.17, 1.18          |
```

Comment:
(Expect) reads odd; either (Expected), or, I think it's fair to assume 1.11 will come out in August 2021.  There's a lot of days left even if the first date is missed :)

Reply:
Removed.